scripts/update-turbo-ios.sh

```diff
@@ -0,0 +1,32 @@
+# First argument is the version tag
+if [ -z "$1" ]
+  then
+    echo "No version tag supplied"
+    exit 1
+fi
+
+# Patch applied to turbo-ios which fixes the resource loading issue
+PATCH_PATH=$(realpath ../patches/turbo-js-resource.patch)
+
+# Update the turbo-ios git submodule
+cd ../packages/turbo/ios/turbo-ios
+git fetch --all --tags --prune
+git checkout tags/$1 -B $1
```

Comment:
Wondering if we should prefix the new branch to be clear that we're using on a "patched" version of the tag?

Reply:
Hm. One issue I'm thinking of here is that the ´ref` that we're pointing to in the submodule, will be non-existent in the `turbo-ios` library without no way to figure which version we're pointing to.